services/server/server.go

```diff
@@ -834,15 +848,24 @@ func (s *Server) PlatformSignUp(ctx context.Context, req *serverpb.PlatformSignU
 	return &serverpb.PlatformSignUpResponse{}, nil
 }
 
-// PlatformSignIn links that PMM instance to Percona Platform user and created new session.
-func (s *Server) PlatformSignIn(ctx context.Context, req *serverpb.PlatformSignInRequest) (*serverpb.PlatformSignInResponse, error) {
+// PlatformConnect connects a PMM server to the organization created on Percona Portal. That allows the user to sign in to the PMM server with their Percona Account.
+func (s *Server) PlatformConnect(ctx context.Context, req *serverpb.PlatformConnectRequest) (*serverpb.PlatformConnectResponse, error) {
 	nCtx, cancel := context.WithTimeout(ctx, platformAPITimeout)
 	defer cancel()
-	if err := s.platformService.SignIn(nCtx, req.Email, req.Password); err != nil {
-		return nil, err
+	if err := s.platformService.Connect(nCtx, req.ServerName, req.Email, req.Password); err != nil {
+		if errors.Is(err, services.ErrAddressNotSet) {
+			return nil, status.Error(codes.FailedPrecondition, err.Error())
```

Comment:
I am not sure if I need this in here. I can set this status.Error in the Connect method. And check only for  `status, ok := status.FromError(err); ok`

Reply:
I fixed it.